gci/models.py

```diff
@@ -1 +1,26 @@
-# Create your models here.
+from django.db import models
+
+class Students(models.Model):
+    identifier = models.IntegerField(default=0, primary_key=True)
+    display_name = models.CharField(max_length=100, default='')
+    email_id = models.EmailField(default='')
+    program_year = models.ForeignKey('Program_year', null=True)
+
+class Organizations(models.Model):
+    org_id = models.IntegerField(default=0, primary_key=True)
+    org_name = models.CharField(max_length=100, default='')
+    org_logo = models.ImageField(upload_to='', blank=True, default='')
+    org_url =  models.URLField(default='', blank=True)
+    org_summary = models.TextField(max_length=300, default='')
+    students = models.ManyToManyField(Students)
+
+class Tasks(models.Model):
+    task_name = models.CharField(max_length=100, default='')
+    languages_req = models.TextField(max_length=200, default='')
+    task_summary = models.TextField(max_length=300, default='')
+    students = models.ForeignKey(Students)
+    _orgs = models.ForeignKey(Organizations)
+
+class Program_year(models.Model):
```

Comment:
`Program_Year` , or `ProgramYear` ?

Reply:
Please use PascalCase of class names, we want it to be consistent so others won't get confused.